.github/workflows/api-checks.yml

```
@@ -0,0 +1,55 @@
+name: Template API Checks
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - app/**
+      - .github/workflows/api-checks.yml
+  pull_request:
+    paths:
+      - app/**
+      - .github/workflows/api-checks.yml
+
+defaults:
+  run:
+    working-directory: ./app
+
+env:
+  # Some commands in the Makefile check if this is set
+  # and add additional parameters to the commands called
+  CI: true
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

not sure the difference but there's a v3 now so might as well use that here and in the rest of the file too

```suggestion
      - uses: actions/checkout@v3
```

---

Looks like just an upgrade to the node version: https://github.com/actions/checkout